app/models.py

```diff
@@ -13,3 +13,5 @@ class Book(models.Model):
     isOnStoreWindow = models.BooleanField(default=False)
     def __str__(self):
         return self.name + ", " + self.author + " (" + self.language + ", " + str(self.releaseDate) + ")"
+    def get_absolute_url(self):
```

Comment:
Wird in keine der Änderungen genutzt?! Falls es in der View genutzt wird, wäre der url-Helper besser geeignet.